### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by DraconicNEO** [CompSciOrBust/SX-Save-Manager#2]
Now that TX is dead do you think there's any chance of getting the Source code released?

**Comment by CompSciOrBust** [comment]
Yes. I have plans for this but I'm too busy with university at the minute. There is a reason why the source can't be released as is and it's complicated by the current situation that TX is in. I hope (but don't promise) that I will be able to release the source by this summer. I would say no sooner than Easter though. The only issue is how much time I have.

**Comment by DraconicNEO** [comment]
> Yes. I have plans for this but I'm too busy with university at the minute. There is a reason why the source can't be released as is and it's complicated by the current situation that TX is in. I hope (but don't promise) that I will be able to release the source by this summer. I would say no sooner than Easter though. The only issue is how much time I have.

That's good. Hopefully that'll be possible, would be nice since it would ensure the project doesn't die if a firmware update breaks it.
